qiskit/visualization/pulse_v2/stylesheet.py

```diff
@@ -196,12 +97,14 @@ class IqxStandard(dict):
     - Plot frame change symbol with formatted operand values.
     - Show chart name with scaling factor.
     - Show snapshot and barrier.
+    - Do not show acquire channels.
     - Channels are sorted by index and control channels are added to the end.
     """
     def __init__(self, **kwargs):
         super().__init__()
         style = {'formatter.control.apply_phase_modulation': True,
                  'formatter.control.show_snapshot_channel': True,
+                 'formatter.control.show_acquire_channel': False,
```

Comment:
I want to feedback from users about this. I always remove them since it doesn't have much information except for some edge cases, for example, tuning of acquisition timing in the context of calibration. but if you want to check them this can be True by default.

Reply:
I'm not sure. @taalexander where do you weigh in on this?

Reply:
We will leave default to True, but certain style sheets should probably set to false. We can open a `discussion` issue on github to get feedback and ideas from users